wiki/Client/Keyboard_shortcuts/de.md

```diff
@@ -1,253 +1,295 @@
+
+#### Spielfeld
+
+| Kürzel | Aktion |
+| :-- | :-- |
+| `Alt` | Schalte beim Halten temporär den [Distance-Snap](/wiki/Client/Beatmap_editor/Distance_snap) ein/aus. |
+| `Y` | Schalte den Distance-Snap ein/aus. |
+| `Shift` | Schalte den Grid-Snap beim Halten ein/aus. |
+| `T` | Schalte den Grid-Snap ein/aus. |
+| `G` | Wechsle durch die Rastergrößen. |
+| `Strg` + `1` | Setze die Rastergröße auf winzig. |
+| `Strg` + `2` | Setze die Rastergröße auf klein. |
+| `Strg` + `3` | Setze die Rastergröße auf mittel. |
+| `Strg` + `4` | Setze die Rastergröße auf groß. |
+| `L` | Schalte die Verriegelung der Hit-Objekte ein/aus. |
+
+#### Objektmanipulierung
+
+| Kürzel | Aktion |
+| :-- | :-- |
+| `Q` | Wechsle zu einer neuen Combo. |
+| `Strg` + `,` | Rotiere die Auswahl um 90 Grad gegen den Uhrzeigersinn. |
+| `Strg` + `.` | Rotiere die Auswahl um 90 Grad mit dem Uhrzeigersinn. |
+| `Strg` + `H` | Spiegle die Auswahl horizontal. |
+| `Strg` + `J` | Spiegle die Auswahl vertikal. |
+| `Strg` + `G` | Mache die Auswahl rückgängig. Das bedeutet, dass das erste ausgewählte Objekt zum letzten ausgewählten Objekt bewegt wird usw. |
+| `Strg` + `Shift` + `R` | Rotiere die ausgewählten Objekte um einen bestimmten Winkel (öffnet einen Dialog). |
+| `Strg` + `Shift` + `S` | Skaliere die ausgewählten Objekte um einen bestimmten Betrag (öffnet einen Dialog). |
+| `Strg` + `Shift` + `D` | Generiere Polygone mit einer bestimmten Anzahl an Punkten (öffnet einen Dialog). |
+| `Strg` + `Shift` + `F` | Konvertiere die ausgewählten Slider zu einem Hit-Circle-Stream (öffnet einen Dialog). |
+| `Strg` + `Pfeil links/rechts/hoch/runter` | Schiebe die ausgewählten Objekte in die entsprechende Richtung. (osu!, osu!taiko, osu!catch) |
+| `Strg` + `Pfeil links/rechts` | Bewege die ausgewählten Objekte eine Spalte nach links oder rechts. (osu!mania) |
+| `Strg` + `Pfeil hoch/runter` | Bewege die ausgewählten Objekte einen Divisor hoch oder runter. (osu!mania) |
 
 #### Hitsounding
 
-- `W`: Whistle-Effekt zu den ausgewählten Objekten hinzufügen
-- `E`: Finish-Effekt zu den ausgewählten Objekten hinzufügen
-- `R`: Clap-Effekt zu den ausgewählten Objekten hinzufügen
-- `Shift` + `Q`, `W`, `E`, `R`: Sampleset für das ausgewählte Objekt ändern zu: Auto, Normal, Soft, oder Drum
-- `Strg` + `Q`, `W`, `E`, `R`: Effekteset für das ausgewählte Objekt ändern zu: Auto, Normal, Soft, oder Drum
-- `Strg` + `Shift` + `I`: Sample-Import öffnen (neues Fenster)
+| Kürzel | Aktion |
+| :-- | :-- |
+| `W` | Füge den Whistle-Effekt zu den ausgewählten Objekten hinzu. |
+| `E` | Füge den Finish-Effekt zu den ausgewählten Objekten hinzu. |
+| `R` | Füge den Clap-Effekt zu den ausgewählten Objekten hinzu. |
+| `Shift` + (`Q`, `W`, `E` oder `R`) | Ändere das Sampleset für das ausgewählte Objekt zu Auto, Normal, Soft bzw. Drum. |
+| `Strg` + (`Q`, `W`, `E` oder `R`) | Ändere das Effekteset für das ausgewählte Objekt zu Auto, Normal, Soft bzw. Drum. |
```

Comment:
I think `oder` sounds better in this context, because its already used in the hotkey column. :thinking: 
```suggestion
| `Shift` + (`Q`, `W`, `E` oder `R`) | Ändere das Sampleset für das ausgewählte Objekt zu Auto, Normal, Soft oder Drum. |
| `Strg` + (`Q`, `W`, `E` oder `R`) | Ändere das Effekteset für das ausgewählte Objekt zu Auto, Normal, Soft oder Drum. |
```

Reply:
The original article uses `respectively` and I tend to translate it with `bzw.`

Reply:
Both are correct anyway, just personal preference so feel free to dismiss this :smile: